tests/sdk/go/helm-get-default-namespace/step2/local-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "local-test-chart.fullname" . }}-new
+  namespace: {{ .Release.Namespace }} # This is what causes the difference when needing to specify a namespace or not during templating.
```

Comment:
This is the helm template format that requires explicit specification of `default` in our resource lookup key.